### ISSUE TITLE
auto clientTrackSwitchOffControl disables tracks with picture-in-picture enabled

### ISSUE THREAD

**Issue by KaelWD** [twilio/twilio-video.js#1550]
- [x] I have verified that the issue occurs with the latest twilio-video.js release and is not marked as a known issue in the [CHANGELOG.md](https://github.com/twilio/twilio-video.js/blob/master/CHANGELOG.md).
 - [x] I reviewed the [Common Issues](https://github.com/twilio/twilio-video.js/blob/master/COMMON_ISSUES.md) and open GitHub issues and verified that this report represents a potentially new issue.
 - [x] I verified that the [Quickstart](https://github.com/twilio/video-quickstart-js) application works in my environment.
 - [x] I am not sharing any [Personally Identifiable Information (PII)](https://www.twilio.com/docs/glossary/what-is-personally-identifiable-information-pii)
       or sensitive account information (API keys, credentials, etc.) when reporting this issue.

**Code to reproduce the issue:**

```js
onVisibilityChange () {
  if (document.hidden) {
    this.startPictureInPicture()
  } else if (document.pictureInPictureElement) {
    document.exitPictureInPicture?.()
  }
},
startPictureInPicture () {
  const el = this.$refs.mainTrack?.$el as HTMLVideoElement | undefined
  if (el && el.readyState >= HTMLMediaElement.HAVE_FUTURE_DATA) {
    el.requestPictureInPicture?.()
  }
},
```

**Expected behavior:**

If the user switches to another tab, the currently active video stream should appear in a PiP window and continue to play. 

**Actual behavior:**

The video appears in a PiP window but is frozen until the tab is active again. 

![image](https://user-images.githubusercontent.com/16421948/129538659-67699292-ce3a-4a60-a369-12e51845676a.png)

**Software versions:**

 - [ ] Browser(s): Chrome 92.0.4515.107
 - [ ] Operating System: Ubuntu 18.04
 - [ ] twilio-video.js: 2.16.0
 - [ ] Third-party libraries (e.g., Angular, React, etc.): vue@2.6.12

**Comment by makarandp0** [comment]
Hello @KaelWD, Thank you for reporting this issue. I see that this is side effect of SDK requesting track switch off in response to tab gaining background. I have filed a bug to track this (VIDEO-6677) 

In the meanwhile you can workaround  this issue by setting `clientTrackSwitchOffControl: "manual"` - This also means that you would have to call [RemoteVideoTrack.switchOff](https://media.twiliocdn.com/sdk/js/video/releases/2.16.0/docs/RemoteVideoTrack.html#switchOff__anchor)  in your application when you do not need to render the track.

Another workaround would be to set internal property `RemoteVideoTrack._enableDocumentVisibilityTurnOff = false` before attaching remote video track. 

Let me know how it goes,
Thanks,
Makarand

**Comment by makarandp0** [comment]
This is now fixed in [2.18.2](https://github.com/twilio/twilio-video.js/releases/tag/2.18.2). Please give it a try and let us know if you see any issues.
